framework/include/utils/MooseTypes.h

```diff
@@ -166,6 +166,8 @@ enum KernelType
 {
   KT_TIME = 0,
   KT_NONTIME = 1,
+  KT_NONEIGEN = 2,
+  KT_EIGEN  = 3,
```

Comment:
I was talking about the constants (0, 1, 2, 3) not the enumerations. I'm just saying that we probably don't need the numbers.